docs/wiki/examples/powershell-inputs/inputs-azure-devops-terraform-microsoft-cloud-for-sovereignty.yaml

```diff
@@ -0,0 +1,85 @@
+---
+# Basic Inputs
+iac: "terraform"
+bootstrap: "alz_azuredevops"
+starter: "microsoft_cloud_for_sovereignty"
+
+# Shared Interface Inputs
+bootstrap_location: ""
+starter_locations: []
+root_parent_management_group_id: ""
+subscription_id_management: ""
+subscription_id_identity: ""
+subscription_id_connectivity: ""
+
+# Bootstrap Inputs
+azure_devops_personal_access_token: ""
+azure_devops_agents_personal_access_token: ""
+azure_devops_organization_name: ""
+use_separate_repository_for_templates: true
+bootstrap_subscription_id: ""
+service_name: "slz"
+environment_name: "mgmt"
+postfix_number: "1"
+azure_devops_use_organisation_legacy_url: false
+azure_devops_create_project: true
+azure_devops_project_name: ""
+use_self_hosted_agents: true
+use_private_networking: true
+allow_storage_access_from_my_ip: false
+apply_approvers: []
+create_branch_policies: true
+apply_alz_archetypes_via_architecture_definition_template: true
+
+# Starter Module Specific Variables
+allowed_locations: []
+allowed_locations_for_confidential_computing: []
+az_firewall_policies_enabled: true
+bastion_outbound_ssh_rdp_ports: ["22", "3389"]
+custom_subnets: {
+  AzureBastionSubnet: {
+    address_prefixes: "10.20.15.0/24",
+    name: "AzureBastionSubnet",
+    networkSecurityGroupId: "",
+    routeTableId: ""
+  },
+  AzureFirewallSubnet: {
+    address_prefixes: "10.20.254.0/24",
+    name: "AzureFirewallSubnet",
+    networkSecurityGroupId: "",
+    routeTableId: ""
+  },
+  GatewaySubnet: {
+    address_prefixes: "10.20.252.0/24",
+    name: "GatewaySubnet",
+    networkSecurityGroupId: "",
+    routeTableId: ""
+  }
+}
+customer: "Country/Region"
+customer_policy_sets: {}
+default_location: ""
```

Comment:
Lets remove this and just use `starter_location` or starter_locations`.

Reply:
@VeronicaSea I'll let you make this change.

Reply:
@azure-lm I discussed with Jared, will not remove the default_location in the current release since we use it from multiple places. And they also asked us to not use starter_location since they have the starter_locations.